main.go

```diff
@@ -13,7 +13,11 @@ import (
 // TODO: think about client timeouts https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/
 func main() {
 	cliArgs := os.Args[1:]
-	config := config.Parse(cliArgs)
+	config, err := config.Parse(cliArgs)
+
+	if err != nil {
+		logrus.WithFields(logrus.Fields{"err": err.Error()}).Fatalf("Could not load config")
```

Comment:
If the only field is an error you can use [logrus.WithError()](https://godoc.org/github.com/sirupsen/logrus#WithError)